src/SprykerSdk/Shared/ComposerConstrainer/ComposerConstrainerConstants.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Shared\ComposerConstrainer;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface ComposerConstrainerConstants
+{
+    /**
+     * Specification:
+     * - Returns a list of configured core namespaces which are used to find used core dependencies.
+     *
+     * @uses \Spryker\Shared\Kernel\KernelConstants::CORE_NAMESPACES
+     *
+     * @api
+     */
+    public const CORE_NAMESPACES = 'CORE_NAMESPACES';
```

Comment:
this is never env specific

Reply:
I know but it is like it is, we have this since the beginning. For some things, we need to have this kind of configuration. What else should we do to have configurations which can be used by any module? Would you like to copy the same configuration into all modules where we need it?

Do you have an alternative for this?

Reply:
What about to make it possible to get the KernelConfig in any module config? With this, we could get rid of these configurations.

ModuleConfig.php:
`$coreNamespaces = $this->getKernelConfig()->getCoreNamesapces()`

Reply:
Should we ask the team?